marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx

```diff
@@ -166,7 +150,26 @@ const ZeroStateExplorer = (props) => {
                       </MLRadio.MLGroup>
                     </div>
                   </Col>
-                  <Col span={12}>
+                </Row>
+                <br />
+                <Row>
+                  <Col span={24}>
+                    <div className={styles.input}>
+                      <Input
+                        style={{width: 700}}
```

Comment:
Can we move the css styles to the associated module.scss file for this component ?

Reply:
Sure, I just pushed commit. Can you please review again.